languages/common-lisp/concepts/rest-parameters/about.md

```diff
@@ -1 +1,7 @@
-TODO: add information on rest-parameters concept
+A rest parameter is designated by the `&rest` lambda list keyword in a lambda list. This parameter will be bound to a value which is a list of all the arguments provided by the caller which is not consumed by other parameters.
```

Comment:
Not sure what the official guidance for these sort of things is, but should we have syntax examples for these things? I think that would be good to have maybe.